src/earn/poolInfoScreen/SafetyCard.tsx

```diff
@@ -40,6 +40,24 @@ function Risk({ risk }: { risk: SafetyRisk }) {
   )
 }
 
+export function TripleBars({ safety, barHeights }: { safety: Safety; barHeights?: number[] }) {
```

Comment:
nit, you can set BAR_HEIGHTS as the default here

```suggestion
export function TripleBars({ safety, barHeights = BAR_HEIGHTS }: { safety: Safety; barHeights?: number[] }) {
```